index.html

```diff
@@ -489,19 +489,20 @@ <h2>The <dfn>ThingDescription</dfn> type</h2>
       [[!WOT-TD]]. It is expected to be
       a <a data-cite="infra#parse-json-bytes-to-a-javascript-value">parsed JSON object</a> that is validated using <a data-cite="WOT-TD#json-schema-for-validation">JSON Schema validation</a>.
     </p>
-    <section> <h3> Fetching a Thing Description</h3>
+    <section> <h3> Requesting a Thing Description</h3>
       <p>
-        Fetching a <a>TD</a> given a URL should be done with an external method, such as the <a data-cite="fetch#fetch-api">Fetch API</a> or a HTTP client library, which offer already standardized options on specifying fetch details.
+        Requesting a <a>TD</a> given a URL should be done with the
+        <a href="#dom-wot-requestthingdescription">requestThingDescription()</a>
+        method.
+        Alternatively, external methods, such as the <a data-cite="fetch#fetch-api">Fetch API</a> or an HTTP client library, can be used.
```

Comment:
```suggestion
        Consumers SHOULD request a <a>TD</a> given a URL using
        the <a href="#dom-wot-requestthingdescription">requestThingDescription()</a>
        method. Alternatively, external methods, such as the <a data-cite="fetch#fetch-api">Fetch API</a> or an HTTP client library, can be used.
```

Reply:
Sounds good! Should the `can be used` also be changed into a `MAY be used` then? Or would that be too much?

Reply:
Call 12-12: 
- @zolkis we can't use normative text for consumers. 
- @relu91 ok I agree